src/components/expired-subscription-modal/index.jsx

```diff
@@ -31,15 +18,21 @@ const ExpiredSubscriptionModal = () => {
       footerNode={(
         <ActionRow>
           <Button
-            onClick={onClickHandler}
+            onClick={() => window.open(customerAgreement?.urlForButtonInModal, '_blank')}
```

Comment:
`Button` can act as a standard hyperlink. We should avoid programmatically opening windows like this, where possible. Treating it as a non-standard hyperlink like this prevents default browser behavior (e.g., showing link previews in the bottom of the browser, being able to right click to copy link, etc.). 

```jsx
<Button href={customerAgreement.urlForButtonInModal}>
  {customerAgreement.buttonLabelInModal}
</Button>
```

Also, sanity check: Is it expected this link would open in a new tab/window vs. navigating to the link within the same tab? Given the only thing offboarded VSF learners can do it see this blocking modal, I'm not sure opening in a new tab/window makes sense since there's not really any reason to return to the Learner Portal once clicked.

That said, if you do still need the `_blank`, I would recommend doing something like the following to ensure the `rel` attribute is also added for security purposes.

```jsx
<Button as={Hyperlink} href={customerAgreement.urlForButtonInModal} target="_blank">
  {customerAgreement.buttonLabelInModal}
</Button>
```

Reply:
It was not mentioned in the work either I needed to open it in the same or a new tab. So I decided to open it up in a new tab if the learner didn't copy his. Coupon Code then he could come back and copy his/her code. But now I have to unlink the user when he/she clicks on `Continue Learning`, so I think opening the new link in the same tab is a better idea.